doc/source/about.rst

```diff
@@ -0,0 +1,51 @@
+About
+=====
+
+Authors
+-------
+
+The Pyndl package was mainly developed by
+`Konstantin Sering <https://github.com/derNarr>`_,
+`Marc Weity <https://github.com/trybnetic>`_,
+`David-Elias Künstle <https://github.com/dekuenstle/>`_
+und `Lennart Schneider <https://github.com/sumny>`_. For the full list of
+contributers have a look at `Github's Contributer summary <https://github.com/quantling/pyndl/contributors>`_.
+
+If this work was helpful in your work, feel free to cite it as
+
+    Konstantin Sering, Marc Weitz, David-Elias Künstle, & Lennart Schneider.
+    (2017). Pyndl: Naive discriminative learning in python.
+    http://doi.org/10.5281/zenodo.597964
+
+If you are using BibTex you may want to use this example BibTex entry::
+
+    @misc{pyndl,
+          author       = {Konstantin Sering and
+                          Marc Weitz and
+                          David-Elias Künstle and
+                          Lennart Schneider},
+          title        = {Pyndl: Naive discriminative learning in python},
+          year         = 2017,
+          doi          = {10.5281/zenodo.597964},
+          url          = {https://doi.org/10.5281/zenodo.597964}
+    }
+
+.. note::
+
+    If you want to cite a specific version, check out the history on
+    `zenodo <https://zenodo.org/search?page=1&size=20&q=conceptrecid:%22597964%22&
+    sort=-version&all_versions=True>`_!
+
+
+Funding
+-------
+Pyndl was partially funded by the Humboldt grant, the ERC advandced grant
+and by the University of Tübingen.
+
+Acknowledgements
+----------------
+This package is build as a python replacement for the R ndl2 package. Some
+ideas on how to build the API and how to efficiently run the Rescorla Wagner
+iterative learning on large text corpora are inspired by the way the ndl2
+package solves this problems. The ndl2 package will be published to github in
+August 2017 and a reference will be placed here.
```

Comment:
Has it already been published? (future / past, truth)

Reply:
Not yet, but it is planned to do so after checking the code for sensible material